clients/imodelhub/src/imodelhub/GlobalEvents.ts

```diff
@@ -6,7 +6,6 @@
  * @module iModelHubClient
  */
 import { ClientRequestContext, GuidString, Logger } from "@bentley/bentleyjs-core";
-import { ContextType } from "@bentley/context-registry-client";
 import { AccessToken, AuthorizedClientRequestContext, ECJsonTypeMap, request, RequestOptions, Response, WsgInstance } from "@bentley/itwin-client";
```

Comment:
@Animagne @amikoliunas @austeja-bentley is there a plan to even keep global events going forward in the client library? If not, I'd like to start removing any references to it so that we don't really need to keep it in mind moving forward.

Reply:
The initial clients (for read and write workflows) should probably not have this functionality due to the concern you've stated - to not confuse 3rd party developers. When we figure out the future of global events we could revisit this question and see where would this functionality fit in in terms of client libraries.